.github/workflows/ocr-tests.yml

```
@@ -0,0 +1,32 @@
+name: OCR Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)